ambry-rest/src/main/java/com.github.ambry.rest/NettyResponseChannel.java

```diff
@@ -776,6 +777,14 @@ public void operationComplete(ChannelFuture future) {
       } else {
         handleChannelWriteFailure(future.cause(), true);
       }
+      long responseAfterWriteProcessingTime = System.currentTimeMillis() - writeFinishTime;
+      long channelWriteTime = writeFinishTime - responseWriteStartTime;
+      nettyMetrics.channelWriteTimeInMs.update(channelWriteTime);
+      nettyMetrics.responseMetadataAfterWriteProcessingTimeInMs.update(responseAfterWriteProcessingTime);
+      if (request != null) {
+        request.getMetricsTracker().nioMetricsTracker.addToResponseProcessingTime(channelWriteTime);
+        request.getMetricsTracker().nioMetricsTracker.addToResponseProcessingTime(responseAfterWriteProcessingTime);
```

Comment:
can we update just once. request.getMetricsTracker().nioMetricsTracker.addToResponseProcessingTime(channelWriteTime + responseAfterWriteProcessingTime);